faunadb-scala/src/test/scala/faunadb/CaseClassSpec.scala

```diff
@@ -0,0 +1,17 @@
+package faunadb
+
+import faunadb.values.Codec
+import org.joda.time.LocalDate
+
+/**
+  * This class only exist to test Codec.caseClass macro compilation without importing `faunadb.values._`
+  */
+class CaseClassSpec {
+
+  case class CaseClass(boolean: Boolean, localDate: LocalDate, string: String, char: Char, byte: Byte,
+                       short: Short, int: Int, long: Long, float: Float, double: Double,
+                       array: Array[Int], seq: Seq[Int], list: List[Int], indexedSeq: IndexedSeq[Int],
+                       option: Option[Int], either: Either[Int, String])
```

Comment:
it's missing a `Map` case because there's a PR pending that includes a map encoder/decoder

Reply:
That's fine. If that PR lands first, we should update this one, and vice versa.